scripts/japicmp.sh

```diff
@@ -34,16 +37,21 @@ if [ -z "$GROUP_ID" ]; then
   GROUP_ID=io.servicetalk
 fi
 GROUP_PATH=$(echo "$GROUP_ID" | tr '.' '/')
-BASEPATH=$HOME/.m2/repository/$GROUP_PATH/
+BASEPATH=$MVN_REPO/$GROUP_PATH/
 
-# All servicetalk modules except: servicetalk-benchmarks, servicetalk-bom, servicetalk-examples, servicetalk-gradle-plugin-internal:
+# All servicetalk modules except:
+# servicetalk-benchmarks, servicetalk-bom, servicetalk-examples, servicetalk-gradle-plugin-internal
 ARTIFACTS="$(ls -d -- */ | grep 'servicetalk-' | sed 's/.$//' | \
```

Comment:
I usually prefer `find` vs `ls` if the names may contain spaces.

Reply:
It's known that the names of modules don't contain spaces